src/components/shared/card.tsx

```diff
@@ -232,11 +241,13 @@ const Card = ({ relatedItem, image }: Props): JSX.Element => {
 
     const lastModified = relatedItem.lastModified?.iso8601;
     const date = lastModified ? relativeFirstPublished(fromNullable(new Date(lastModified))) : null;
+    const starRating = relatedItem.starRating ? stars(parseInt(relatedItem.starRating)) : null;
```

Comment:
If this parses to `NaN` I think you'll get a star rating of zero - is that what you expected?